src/components/Home/Hero.js

```diff
@@ -49,7 +49,7 @@ export default function Hero() {
                         <span className="inline-block">Product OS</span>
                     </h1>
                     <h2 className={heading('subtitle', 'primary', 'my-6 !text-black/50')}>
-                        A suite of product and data tools. Built on the modern data stack.
+                        PostHog integrates analytics, session recording, feature flags, and more - all into one seamless platform. Save time configuring multiple tools, and more time building better products.   
```

Comment:
```suggestion
                        Analytics, session recording, feature flags and more. All in one platform.   
```
```suggestion
                        Analytics, recordings, feature flags and more. Spend less time switching tools, more time shipping.  
```

We can probably trim down to something like this?

Reply:
I think the all-in-oneness though is still our killer USP? Yes we're splitting pricing, but the advantage of PostHog over our competitors is our integration between these tools - we don't win when you compare these features alone 1 on 1 most of the time.